config/environments/production.rb

```diff
@@ -56,20 +56,24 @@
   # when problems arise.
   config.log_level = :info
 
-  # Prepend all log lines with the following tags.
-  config.log_tags = [
-    :uuid,
-    proc do |request|
-      if request.headers['Authorization'] =~ /Token token=(.*)/
-        Session.obscure_token(Regexp.last_match[1])
-      else
-        'unauthenticated'
-      end
+  config.log_tags = {
+    request_id: :uuid,
+    session: proc do |request|
+      Session.obscure_token(Regexp.last_match[1]) if request.headers['Authorization'] =~ /Token token=(.*)/
     end,
-    proc do |_request|
-      AppInfo::GIT_REVISION
-    end
-  ]
+    ref: ->(_request) { AppInfo::GIT_REVISION },
+    consumer_id: ->(request) { request.headers['X-Consumer-ID'] },
```

Comment:
Not sure we want/need all three of these, but I don't know which we'd want yet - depends on if we are using Kong to generate IDs or have an external system generate IDs that we plug in to Kong.

Reply:
👍 doesn't hurt (much) to have the additional information included for now, since it's not ordered like it was without structured logging.